Library/Formula/freeswitch.rb

```diff
@@ -146,8 +149,27 @@ def install
     system "./configure", "--disable-dependency-tracking",
                           "--enable-shared",
                           "--enable-static",
-                          "--prefix=#{prefix}",
-                          "--exec_prefix=#{prefix}"
+                          "--prefix=#{etc}/freeswitch",
+                          "--exec_prefix=#{prefix}",
+                          "--with-htdocsdir=#{doc}/htdocs",
+                          "--includedir=#{include}",
+                          "--with-modinstdir=#{prefix}/mod",
+                          "--with-logfiledir=#{var}/log/#{name}",
+                          "--localstatedir=#{var}/#{name}",
+                          "--with-rundir=#{var}/run/#{name}",
+                          "--with-dbdir=#{var}/#{name}/db",
+                          "--with-soundsdir=#{var}/#{name}/sounds",
+                          "--with-recordingsdir=#{var}/#{name}/recordings",
+                          "--with-storagedir=#{var}/#{name}/storage",
+                          "--with-cachedir=#{var}/#{name}/cache"
+
+    if build.with?("flite")
+      inreplace "modules.conf", "#asr_tts/mod_flite", "asr_tts/mod_flite"
```

Comment:
I had the CI server build mod_flite & mod_directory previously (by making them opt-out) and it worked fine. Since you are looking at a newer version of freeswitch maybe something has changed. But this does seem to work for 1.4.23.